### ISSUE TITLE
struggling bad

### ISSUE THREAD

**Issue by xanxx20** [EvernodeXRPL/evernode-host#158]
ive been at this for 4 days now ive tried multiple hosts and vps
i finally came accross a guide : https://medium.com/@ilaNihas/evernode-become-a-pioneering-host-9ef88a0756d4
with no luck... most of everything connected. except for the cert bot, i got close so many times but idk how this is not working step for step.
i know it says false records but it wasnt even saying a valid domain without connecting my dns to my inet ip
while the guide above said to run it through the vps.

ive gotten countless logs but this is the most recent:

---Sashimono prerequisites installer---
STAGE Installing dependencies
Hit:1 https://deb.nodesource.com/node_20.x nodistro InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu focal InRelease
Hit:3 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:4 http://security.ubuntu.com/ubuntu focal-security InRelease
Hit:5 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
quota is already the newest version (4.05-1).
cgroup-tools is already the newest version (0.41-10).
fuse3 is already the newest version (3.9.0-2).
curl is already the newest version (7.68.0-1ubuntu2.21).
openssl is already the newest version (1.1.1f-1ubuntu2.20).
uidmap is already the newest version (1:4.8.1-1ubuntu5.20.04.4).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up fuse3 (3.9.0-2) ...

Configuration file '/etc/fuse.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** fuse.conf (Y/I/N/O/D/Z) [default=N] ? dpkg: error processing package fuse3 (--configure):
 end of file on stdin at conffile prompt
Errors were encountered while processing:
 fuse3
E: Sub-process /usr/bin/dpkg returned an error code (1)
STAGE Configuring fstab
fstab already configured.
STAGE Configuring fuse
Fuse config already updated.
STAGE Configuring cgroup rules engine
STAGE Configuring grub
Grub already configured.
---Sashimono installer---
STAGE Setting up letsencrypt certbot
core 16-2.60.4 from Canonical** installed
snap "core" has no updates available
certbot 2.8.0 from Certbot Project (certbot-eff**) installed
Rules updated
Rules updated (v6)
Running certbot certonly
Account registered.
Requesting a certificate for xrpisatacoshell.xyz

Certbot failed to authenticate some domains (authenticator: standalone). The Certificate Authority reported these problems:
  Domain: xrpisatacoshell.xyz
  Type:   dns
  Detail: no valid A records found for xrpisatacoshell.xyz; no valid AAAA records found for xrpisatacoshell.xyz

Hint: The Certificate Authority failed to download the challenge files from the temporary standalone webserver started by Certbot on port 80. Ensure that the listed domains point to this machine and that it can accept inbound connections from the internet.

Saving debug log to /var/log/letsencrypt/letsencrypt.log
Some challenges have failed.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.
Error when setting up letsencrypt SSL certificate.
Rolling back sashimono installation.
---Sashimono uninstaller---
Removing Sashimono cgroup creation service...
Failed to stop sashimono-cgcreate.service: Unit sashimono-cgcreate.service not loaded.
Failed to disable unit: Unit file sashimono-cgcreate.service does not exist.
rm: cannot remove '/etc/systemd/system/sashimono-cgcreate.service': No such file or directory
Removing Sashimono service...
Failed to stop sashimono-agent.service: Unit sashimono-agent.service not loaded.
Failed to disable unit: Unit file sashimono-agent.service does not exist.
rm: cannot remove '/etc/systemd/system/sashimono-agent.service': No such file or directory
Removing Sashimono private docker registry...
sashidockerreg does not exist.
Deleting binaries...
Deleting Sashimono CLI...
rm: cannot remove '/usr/bin/sashi': No such file or directory
Deleting cgroup rules...
Restarting the 'cgrulesengd' service...
groupdel: group 'sashiuser' does not exist
Deleting message board user...
Deleting binaries...
Deleting data directory...
Sashimono uninstalled successfully.
Rolled back the installation.

**Comment by xanxx20** [comment]
i have a sub domain, but im not exactly sure on what configs i need to set between my vps, inet, and domain dns

**Comment by xanxx20** [comment]
yes it points to my vms ip and i replaced the orignal ip that was on my domain with my vms. and my subdomain has my vps.




> Can you check from an online DNS checker wether xrpisatacoshell.xyz points to your VM's IP. If so can you also check wether if there are any web servers running on port 80?

just did eveything looks good, port 80 is clear i removed the websrv

**Comment by chalith** [comment]
Can you also censure there's no firewall rules for port 80 inbound connections?
